### PR TITLE
An example to deploy nebula in a kubernetes cluster and access it from localhost

### PR DESCRIPTION
**Pull request by wyuan1704**

### CODE REVIEW

File: docs/docs/intro.md [hunk 1]
```diff
@@ -82,4 +82,87 @@ For details, please see [Nebula Discovery](basics/5-discovery.md)
...
+kubectl apply -f nebula.yaml
+```
+
+* Use port forwarding to view nebula UI from your browser.
+
+```shell
+kubectl port-forward service/nebula 8088:8088
+```
+
+Now open http://localhost:8088 to play with nebula UI. You can scale up by simply increasing "replicas" of the deplyment nebula-node in nebula.yaml and running `kubectl apply -f nebula.yaml` again.
```
Comment (shawncao) [inline docs/docs/intro.md:168]: Thanks a ton! This is great! 

I just followed this instruction to run these commands on my local box, the nebula web and nebula server seems working fine at http://localhost:8088, I can see web UI and aI can see table list too, but failed to execute query, seems nebula server can't reach nebula nodes, do you know what might go wrong?

File: docs/docs/intro.md [hunk 2]
```diff
@@ -82,4 +82,87 @@ For details, please see [Nebula Discovery](basics/5-discovery.md)
...
+    metadata:
+      labels:
+        app: nebula-node
+    spec:
+      containers:
+        - name: nebula-node
+          image: columns/nebula.node
+          env:
+          - name: NSERVER
+            value: "nebula:9190"
```
Comment (shawncao) [inline docs/docs/intro.md:152]: should this be `nebula-server:9190` or just `nebula:9190`?

**Review by shawncao — APPROVED** [review]
Thanks Jason -  I need you help figured out why query execution failed for me...

File: docs/docs/intro.md (hunk 2 above)
Comment (wyuan1704) [reply docs/docs/intro.md:152]: This is right. See line 119 that the name of service is "nebula"

File: docs/docs/intro.md (hunk 1 above)
Comment (wyuan1704) [reply docs/docs/intro.md:168]: Does it show that "nebula is down"? I can do some queries.

When I run `kubectl logs -l app=nebula-server -c nebula-server --tail=-1`, I can see `NodeSync.cpp:176] Communicated tasks=1 to nodes=2 using ms=3`, which means that they are connected, right?

Comment (shawncao) [reply docs/docs/intro.md:168]: yes, interesting, seems like it connects at the beginning, and the data loading was done correctly, but time out in query time, this is what I see using your command:

```
I0916 00:14:41.259738     1 SpecRepo.cpp:323] Updating 1 specs: brandnew=1, renew=0, removed=18446744073709551615, count=1
I0916 00:14:41.261327     1 ConnectionPool.cpp:64] Creating a channel to 10.1.0.7:9199
I0916 00:14:41.299954     1 ConnectionPool.cpp:64] Creating a channel to 10.1.0.8:9199
I0916 00:14:41.322290     1 NodeSync.cpp:176] Communicated tasks=1 to nodes=2 using ms=64
I0916 00:14:46.698388    12 NebulaServer.cpp:123] Served table list request.
I0916 00:14:51.344208     1 NodeSync.cpp:176] Communicated tasks=1 to nodes=2 using ms=18
I0916 00:15:28.624260    13 NebulaServer.cpp:123] Served table list request.
I0916 00:15:28.644600    12 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:15:28.648694    12 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:15:34.543217    12 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:15:34.543431    12 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:15:34.595127    12 Dsl.cpp:189] Mask a column due to access policy: event
I0916 00:15:34.595636    12 Dsl.cpp:226] Query output schema: STRUCT<_time_:BIGINT,_time_:BIGINT,event:VARCHAR,flag:BOOLEAN,id:INTEGER,stack:VARCHAR,stamp:INT128,tag:VARCHAR,value:TINYINT,weight:DOUBLE> w/ agg columns: 0
I0916 00:15:34.601227    12 Dsl.cpp:354] Nodes to execute the query: 1
I0916 00:16:00.850220    20 NebulaServer.cpp:123] Served table list request.
I0916 00:16:00.867642    20 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:00.867970    20 NebulaServer.cpp:164] Served table stats request for nebula.test
W0916 00:16:14.702461    21 ServerExecutor.cpp:56] RPC Timeout: 35000
I0916 00:16:15.309831    13 NebulaServer.cpp:123] Served table list request.
I0916 00:16:15.333962    20 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:15.334364    20 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:16:16.505080    20 NebulaServer.cpp:123] Served table list request.
I0916 00:16:16.518460    13 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:16.518697    13 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:16:17.092070    20 NebulaServer.cpp:123] Served table list request.
I0916 00:16:17.102283    13 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:17.102466    13 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:16:17.293503    13 NebulaServer.cpp:123] Served table list request.
I0916 00:16:17.306555    13 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:17.306748    13 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:16:18.273524    20 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:16:18.273711    20 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:16:18.297472    20 Dsl.cpp:189] Mask a column due to access policy: event
I0916 00:16:18.297618    20 Dsl.cpp:226] Query output schema: STRUCT<_time_:BIGINT,_time_:BIGINT,event:VARCHAR,flag:BOOLEAN,id:INTEGER,stack:VARCHAR,stamp:INT128,tag:VARCHAR,value:TINYINT,weight:DOUBLE> w/ agg columns: 0
I0916 00:16:18.297964    20 Dsl.cpp:354] Nodes to execute the query: 1
W0916 00:17:00.158200    21 ServerExecutor.cpp:56] RPC Timeout: 35000
I0916 00:17:26.572367    13 NebulaServer.cpp:123] Served table list request.
I0916 00:17:26.589125    23 NebulaServer.cpp:129] Look up state for table nebula.test
I0916 00:17:26.589341    23 NebulaServer.cpp:164] Served table stats request for nebula.test
I0916 00:17:26.624253    23 Dsl.cpp:189] Mask a column due to access policy: event
I0916 00:17:26.624380    23 Dsl.cpp:226] Query output schema: STRUCT<_time_:BIGINT,_time_:BIGINT,event:VARCHAR,flag:BOOLEAN,id:INTEGER,stack:VARCHAR,stamp:INT128,tag:VARCHAR,value:TINYINT,weight:DOUBLE> w/ agg columns: 0
I0916 00:17:26.624621    23 Dsl.cpp:354] Nodes to execute the query: 1
W0916 00:18:08.634220    21 ServerExecutor.cpp:56] RPC Timeout: 35000
```

Comment (shawncao) [reply docs/docs/intro.md:168]: I see the query sent to Node, it seems to me the Node failed to execute it, wondering if some new runtime error with the new docker image.

So it's related to your config example, we can merge the change:
command `kubectl logs -l app=nebula-node -c nebula-node --tail=-1`

```
I0916 00:17:26.629418    15 Dsl.cpp:354] Nodes to execute the query: 1
I0916 01:59:01.528224    17 Dsl.cpp:226] Query output schema: STRUCT<_time_:BIGINT,_time_:BIGINT,event:VARCHAR,flag:BOOLEAN,id:INTEGER,stack:VARCHAR,stamp:INT128,tag:VARCHAR,value:TINYINT,weight:DOUBLE> w/ agg columns: 0
I0916 01:59:01.531994    17 Dsl.cpp:354] Nodes to execute the query: 1
```

Comment (wyuan1704) [reply docs/docs/intro.md:168]: For the nubula.node logs, you can use `kubectl logs -l app=nebula-node` to get all 2 nodes logs or `kubectl logs nebula-node' (press tab to get pod names) to see one nebula-node logs. BTW, `kubectl get pods' can get all pods.

You can also try to decrease the "replicas" field to 1 and run `kubectl apply -f nebula.yaml` to simplify the debugging.

**Comment by shawncao** [comment]
Oh, this is cool, thanks for the tip!

**Pull request merged**